classes/04class/exercises/c04-iac04/nimmi89/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "image_id" {
```

Comment:
This file is duplicated, should only be inside `terraform-code` folder. Please remove it.

Reply:
Removed!